go.mod

```diff
@@ -1,3 +1,8 @@
-module github.com/sensu-community/check-plugin-template
+module github.com/{{ .GithubUser }}/{{ .GithubProject }}
 
 go 1.13
+
+require (
+	github.com/sensu-community/sensu-plugin-sdk v0.5.0-alpha3
```

Comment:
There are newer versions of `sensu-plugin-sdk` available; should we update that too?